modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,126 @@
+= Configure Teams
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Every root organization has a main _parent team_ that appears by default. Every user in your organization is a member of the highest parent team and receives all permissions therein.
+You can create _child teams_ under the parent team and configure those teams to have different sets of permissions.
+After you create child teams and assign permissions, you can add users to each of your teams according to what permissions the users should have.
+Further, you can assign _team maintainers_ who can modify users and permissions within the teams they maintain.
+
+Users who have the Organization Administrator permission can also change the name of a team, move child teams under different parent teams, or delete teams.
+
+== Create a Child Team
+
+If you have the Organization Administrator permission or are a team maintainer, you can create child teams under a parent team.
+The child team inherits permissions from its parent team, but you configure additional permissions for its users.
+
+To create a child team in the Teams feature interface:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. In the Access Management navigation menu, click *Teams*.
+. Select *Create Team*.
+. Enter a unique and intuitive name for your child team. +
+You can use only alphanumeric characters, hyphens, commas, periods, and spaces. The character limit is 255.
++
+. Select a parent team.
+. Select *Save*.
+
+The child team appears under its parent team in the list of teams. You can also select a parent team and navigate to the *Child teams* tab to see a list of all child teams under that parent team.
+
+== Manage Team Permissions
+
+When you begin using the Teams feature, your organization’s main team is named “Everyone at <Your Organization>” and does not have any permissions assigned.
+If you have the Organization Administrator permission, you can configure permissions for this team in addition to child teams.
+If you invite users to “Everyone at <Your Organization>” without assigning any permissions to that team, your users cannot perform execute tasks or view assets.
```

Comment:
What does this line mean?

Reply:
It'll be the same as today if you invite a user without assigning them any role groups. I think they get access to the Exchange public portal and some parts of API Manager?